app/controllers/student_images_controller.rb

```diff
@@ -1,18 +1,23 @@
 # frozen_string_literal: true
 
 class StudentImagesController < ApplicationController
+  skip_after_action :verify_policy_scoped, only: :index
```

Comment:
No point in scoping student images. If a user can access a student, they can see all of their images.